<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextSearch</class>
 <widget class="QMainWindow" name="TextSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="directoryLabel">
      <property name="text">
       <string>Каталог с текстовыми файлами:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="directoryLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Обзор...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="searchLabel">
      <property name="text">
       <string>Искомая фраза:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="threadLabel">
      <property name="text">
       <string>Количество потоков:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="threadSpinBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Начать поиск</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="resultTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>122</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Имя файла</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество вхождений</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
